### PR TITLE
WDIO update to 8.23.0

### PR DESCRIPTION
**Pull request by jperez1-via**
# PR Details

First iteration with set of changes that allow to execute beacon related e2e tests with WDIO v8

## Description

Created lib version 2.0.0 with updated dependencies and latest version of related WDIO v8 packages. Adapted tsconfig to be ESM compatible and adapted imports and other methods for WDIO v8. 

## Related Issue

#39 

## Motivation and Context

Keep library up to date by using latest version of dependencies

## How Has This Been Tested

This has been tested by using it within the Beacon project. E2E tests execute successfully although it has been detected already some related tasks that will need to be done -> refactor of test-case.reportes.ts. Other changes might be needed.

## Types of changes

- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x ] I have read the **CONTRIBUTING** document
- [x ] My code follows the code style of this project
- [x ] Increase version in package.json following [Semantic Versioning](https://semver.org/)
- [x ] Add the issue into the right [project](https://github.com/systelab/systelab-components-wdio-test/projects) with the proper status (In progress)

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -3,18 +3,19 @@
...
         "sourceMap": false,
-        "target": "es5",
-        "module": "commonjs",
-        "moduleResolution": "node",
+        "target": "ES2020",
+        "module": "ESNext",
+        "moduleResolution": "Node",
         "importHelpers": true,
         "removeComments": true,
-        "noImplicitAny": true,
```
Comment (joaquimvila) [inline tsconfig.json:11]: This `noImplicitAny` option is now less strict. Is this a must for running with WDIO v8? If it isn't, I would re-enable this option to force us to have a more typed code in this library.

File: tsconfig.json [hunk 2]
```diff
@@ -3,18 +3,19 @@
...
+        "noImplicitAny": false,
         "strictPropertyInitialization": true,
         "strictNullChecks": true,
         "esModuleInterop": true,
+        "skipLibCheck": false,
```
Comment (joaquimvila) [inline tsconfig.json:15]: Is this needed? Isn't the default already set to false?

**Review by joaquimvila — APPROVED** [review]
Good job @jperez1-via with this upgrade! Just added a couple of questions to double-check TypeScript settings are the right ones.

**Review by Franek75 — APPROVED** [review]

**Review by atzurk — APPROVED** [review]
This is a great upgrade, I've been struggling with the test-case.reporter class to make it compatible in typescript but at no avail... very good job!!

File: tsconfig.json (hunk 2 above)
Comment (jperez1-via) [reply tsconfig.json]: Set it to true, as there are currently some collateral issues when building the lib. However, lib works ok even outputing these errors

**Comment by jperez1-via** [comment]
Created systelab-components-wdio-test v2.0.0 base wtih WDIO v8 - completed

**Pull request merged**